src/lib.rs

```diff
@@ -60,6 +60,10 @@
 pub use log::Level;
 pub use log::LevelFilter;
 
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
```suggestion
#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Reply:
This is perfect. I'm new to rust and miss some of these obvious, more elegant solutions. Thank you.